packages/test-helpers/src/withComponent.js

```diff
@@ -1,12 +1,19 @@
 /* globals describe */
 import log from "npmlog";
+import assert from "assert";
+import path from "path";
+import stack from "callsite";
 import Client from "../../client";
+import AppSnaps from "../../snapshots";
 import rnComponentKey from "../../common/rnComponentKey";
 
 const client = Client(7811);
 log.verbose("client socket", client.socket);
 export default () => {
   const withComponent = (component, description, tests) => {
+    const testfilePath = stack()[1].getFileName();
```

Comment:
could we change these variable names to something more meaningful

Reply:
that is a meaningful name - we are getting the path of the test file